src/Symfony/Component/Serializer/Normalizer/ConstraintViolationListNormalizer.php

```diff
@@ -43,6 +44,11 @@ public function __construct($defaultContext = [], NameConverterInterface $nameCo
      */
     public function normalize($object, string $format = null, array $context = [])
     {
+        $payloadFieldsToSerialize = $context[self::PAYLOAD_FIELDS] ?? $context[self::PAYLOAD_FIELDS] ?? null;
```

Comment:
Not necessary to use the null coalescing operator twice.

Reply:
Good catch, fixed (the second one must be the default context)